nav2_costmap_2d/CMakeLists.txt

```diff
@@ -27,6 +27,13 @@ find_package(angles REQUIRED)
 remove_definitions(-DDISABLE_LIBUSB-1.0)
 find_package(Eigen3 REQUIRED)
 
+find_package(OpenMP)
```

Comment:
inflation_tests.missing_result
whitespace/extra (CMakeLists.txt:31)
whitespace/extra (CMakeLists.txt:32)
whitespace/extra (CMakeLists.txt:33)
whitespace/extra (CMakeLists.txt:34)

You have some linting issues according to CI using the ament cmake linter (there's some other test failures that aren't your fault, I just haven't had time to fix the other linting issues yet).